One test fails because the expected string contains the reassigned values

```
Test text: <p>Your should use the correct story format for first story: <code>"Once upon a time, there was a(n) [adjective] [noun] who loved to eat [noun2]. The [noun] lived in a [place] and had [adjective2] nostrils that blew fire when it was [verb]."</code>. Pay attention to spaces.</p>
      + expected - actual

      -Once upon a time, there was a(n) funny dragon who loved to eat cakes. The dragon lived in a garden and had sparkling nostrils that blew fire when it was jumping.
      +Once upon a time, there was a(n) cute puppy who loved to eat flower. The puppy lived in a park and had colorful nostrils that blew fire when it was barking.
      
  AssertionError: expected 'Once upon a time, there was a(n) funn…' to equal 'Once upon a time, there was a(n) cute…'
```

--- a/curriculum/challenges/english/25-front-end-development/lab-sentence-maker/66c057041df6394ca796bf33.md
+++ b/curriculum/challenges/english/25-front-end-development/lab-sentence-maker/66c057041df6394ca796bf33.md
@@ -122,7 +122,14 @@ assert.isNotNull(firstStory);
 You should use the correct story format for the first story: `"Once upon a time, there was a(n) [adjective] [noun] who loved to eat [noun2]. The [noun] lived in a [place] and had [adjective2] nostrils that blew fire when it was [verb]."`. Pay attention to spaces.
 
 ```js
-const expected = `Once upon a time, there was a(n) ${adjective} ${noun} who loved to eat ${noun2}. The ${noun} lived in a ${place} and had ${adjective2} nostrils that blew fire when it was ${verb}.`;
+const _initialValues = {}
+for (const name of ['adjective', 'noun', 'noun2', 'place', 'verb', 'adjective2']) {
+    const match = code.match(new RegExp(String.raw`${name}\s*=\s*('|"|${'`'})(?<${name}>.*)\1;?\s*`));
+    _initialValues[name] = match ? match.groups[name] : null;
+}
+
+const expected = `Once upon a time, there was a(n) ${_initialValues['adjective']} ${_initialValues['noun']} who loved to eat ${_initialValues['noun2']}. The ${_initialValues['noun']} lived in a ${_initialValues['place']} and had ${_initialValues['adjective2']} nostrils that blew fire when it was ${_initialValues['verb']}.`;
+
 assert.strictEqual(firstStory, expected);
 ```
 

